src/test/java/edu/harvard/iq/dataverse/export/ddi/dataset-finchDC.xml

```diff
@@ -18,4 +18,5 @@
   <dcterms:spatial>USA</dcterms:spatial>
   <dcterms:spatial>MA</dcterms:spatial>
   <dcterms:spatial>Other Geographic Coverage</dcterms:spatial>
+  <dcterms:license>CC0-1.0</dcterms:license>
```

Comment:
Hmm, should the SWORD API page be updated? Yes, from a quick look, it should be. Even better, for backward compatibility, can we continue to support "CC0" instead of requiring "CC0-1.0" (which I assume is the case).

Reply:
Yes the docs should match. I guess it's a good question whether to stay backward compatible. Google's search recommendation (talking more about URLs than names) recommends including the specific version number. Whether this is a good time to adopt that convention, or even to adopt the URL form here instead, is a good question.

At a minimum, keeping the examples, tests, and docs in sync is good and we also need to keep track of whether the name of CC0  changes to 'CC0 Waiver' etc. as discussed above which would affect this as well.